examples/custom-roles-analyzer/docs/code-of-conduct.md

```diff
@@ -0,0 +1,93 @@
+# Code of Conduct
```

Comment:
I think we should move this file to the root of the repo so it applies to the all the code not just this tool

Reply:
Added this file at root level.